### PR TITLE
feat: Add new navigation menu style

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const routes = {
   '/:dash': Dash.Overview,
 
   '/:dash/apps': Dash.Apps,
+  '/dev/apps': Dash.Apps,
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/routes/index.js]: Apps list page , admin has secondary menu, developer no need. Judge whether has  secondary menu according to route path.

File: src/components/Layout/SideNav/index.jsx [hunk 2]
```diff
@@ -67,48 +83,95 @@ class SideNav extends React.Component {
...
-                  size={nav.iconName === 'op-logo' ? 16 : 24}
-                  type="white"
-                />
+      <div className={styles.nav}>
+        <ul className={styles.topNav}>
+          <li>
+            <Link to="/">
+              <img src="/logo_icon.svg" className={styles.icon} />
+            </Link>
+            <label className={styles.title}>QingCloud 应用中心</label>
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: ok

File: src/components/Layout/TitleBanner.jsx [hunk 3]
```diff
@@ -3,15 +3,21 @@ import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
 import { observer } from 'mobx-react';
 import { translate } from 'react-i18next';
+import { inject } from 'mobx-react/index';
```
Comment (iwisunny) [inline src/components/Layout/TitleBanner.jsx]: No need from `mobx-react/index`, just
```
import {observer, inject} from 'mobx-react'
```

File: src/components/Layout/TitleBanner.jsx [hunk 4]
```diff
@@ -33,22 +39,63 @@ class TitleBanner extends Component {
...
+        detail = appStore.appDetail;
+        hasImage = true;
+        break;
+      case 'clusterDetail':
+        detail = clusterDetailStore.cluster;
+        break;
+      case 'runtimeDetail':
+        detail = runtimeStore.runtimeDetail;
+        break;
+    }
```
Comment (iwisunny) [inline src/components/Layout/TitleBanner.jsx]: Add default break to switch statement.

File: src/components/Layout/SideNav/index.jsx [hunk 5]
```diff
@@ -67,48 +83,95 @@ class SideNav extends React.Component {
...
-  renderNav(role) {
-    const { t } = this.props;
-
-    let navs = getNavs(role);
+  renderNav() {
+    const { user, appStore, history, t } = this.props;
+    const { role, isDev } = user;
+    let navs = getNavs[role];
+    const { menuApps } = appStore;
+    let bottomNavs = getBottomNavs;
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: next code:  bottomNavs = bottomNavs.slice(2);

File: src/components/Layout/TitleBanner.jsx (hunk 4 above)
Comment (whDongRui) [reply src/components/Layout/TitleBanner.jsx]: ok

File: src/components/Layout/TitleBanner.jsx (hunk 3 above)
Comment (whDongRui) [reply src/components/Layout/TitleBanner.jsx]: ok.

File: src/components/Layout/SideNav/index.jsx [hunk 6]
```diff
@@ -7,13 +7,24 @@ import classnames from 'classnames';
...
 import MenuLayer from 'components/MenuLayer';
 
-import { subNavMap, getNavs } from './navMap';
+import { subNavMap, getNavs, getDevSubNavs, getBottomNavs } from './navMap';
 
 import styles from './index.scss';
 
-const keys = ['app', 'review', 'cluster', 'runtime', 'repo', 'categories', 'category', 'user'];
+const keys = [
+  'app',
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/components/Layout/SideNav/index.jsx]: There no blank line?

File: src/routes/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iwisunny) [reply src/routes/index.js]: ok

File: src/components/Layout/SideNav/index.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (iwisunny) [reply src/components/Layout/SideNav/index.jsx]: Caused by github split window

**Pull request merged**